public/src/actions/campaign.jsx

```diff
@@ -59,3 +66,12 @@ export function fetchCampaigns(status = '') {
     console.log('error fetching all campaigns from the db: ', err);
   });
 }
+
+export function verifyVolunteerCampaign(userId, campaignId) {
+  return dispatch => axios.get(`/users/${userId}/campaigns/${campaignId}`)
+    .then(response => dispatch(setUserCampaignJoin(response)))
```

Comment:
Right now, the controller is just sending a status without a body. Would it make sense to get the response.statusCode and setUserCampaignJoin according to whether it comes back as 204 or 404?

Reply:
Pretty sure V mentioned that a 204 would hit the .then block as a defined promise value, but I'll check in with him to make sure this is following the ticket specs. :)

Reply:
Can you update with what he said in that checkin?